aqua/qt5/Portfile

```
@@ -1867,7 +1867,9 @@ foreach {module module_info} [array get modules] {
                 #    for linking.
                 # We must choose one or the other.
                 # see https://trac.macports.org/ticket/54592
-                patchfiles-append patch-qtmultimedia_no_macports_openal.diff
+                patchfiles-append \
+                    patch-qtmultimedia_no_macports_openal.diff \
+                    patch-qtmultimedia_no_std_unary_function.diff
```

This `patchfiles-append` already has a comment not relevant to the patch being added, so I suggest not reusing it:
```suggestion
                patchfiles-append patch-qtmultimedia_no_macports_openal.diff

                # std::unary_function removed in Xcode clang 15
                patchfiles-append patch-qtmultimedia_no_std_unary_function.diff
```